libretroshare/src/retroshare/rsgxscircles.h

```diff
@@ -95,17 +94,17 @@ struct RsGxsCircleGroup : RsSerializable
 		RS_SERIAL_PROCESS(mInvitedMembers);
 		RS_SERIAL_PROCESS(mSubCircles);
 	}
+
+	~RsGxsCircleGroup() override;
```

Comment:
Why move this to the .cc ?? When I read the .h it lets me think that something special is done in the destructor whereas it's not.

Reply:
It is to avoid the compiler warning I talked to you some time ago, about the vtable being emitted in every compilation unit when no method is defined outside the .h where the class is declared, moving the default destructor outside is the less invasive thing we can do to avoid that